src/app/dynamic-forms/fields/child-form-table/child-form-table.component.html

```diff
@@ -49,7 +49,13 @@
       </tbody>
     </table>
   </div>
-  <p *ngIf="!childForm.childFormRowValues?.length">
-    There is no data in this table. You can use the add button above to get started.
+  <p *ngIf="childForm.childFormLinkId == null || childForm.childFormParentId == null ;else noRecord"> 
```

Comment:
This isn't a valid condition. The purpose of this message was to show an empty state for when there are no records to display, but in that case you're still able to create new entries. If the child form isn't configured correctly (which is the condition you're checking) then you need to either show something else or just not render the entire form

Reply:
@DAquilinaCinchy FYI, I can't comment on the code but this appears to work exactly as expected. If the criteria is met (required columns have values) then the "There is no data in this table. You can use the add button above to get started" is displayed. If not, "The columns "Child Form Link Field" or both the “Child Form Parent ID” and the "Child Form Link ID" must be populated in your form fields table." is displayed.